test/actions/profile.spec.js

```diff
@@ -53,7 +53,7 @@ describe('deleteComment', () => {
     expect(typeof deleteComment).toBe('function');
   });
 
-  it('should create an appropiate action', () => {
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 

```suggestion
  it('should create an action', () => {
```